app/sequencescape_excel/sequencescape_excel/specialised_field/dual_index_tag_well.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module SequencescapeExcel
+  module SpecialisedField
+    ##
+    # DualIndexTagWell
+    class DualIndexTagWell
+      # 2 tags in the well, 1 tag from tag_group and 1 tag from tag2_group
+      TAGS_PER_WELL = 2
+
+      include Base
+      include ValueRequired
+      include ValueToUpcase
+
+      attr_accessor :sf_dual_index_tag_set
+
+      validates :well_index, presence: { message: 'is not valid' }
+      validates :tags, length: { is: TAGS_PER_WELL, message: 'does not have associated tags' }, if: :well_index
+
+      def update(_attributes = {})
+        return unless valid?
+
+        raise StandardError, 'Tag aliquot mismatch' unless asset.aliquots.one?
+
+        asset.aliquots.first.update(tag: tags.first, tag2: tags.second)
```

Comment:
I assume this works because the query `Tag.where(tag_group_id: [tag_group_id, tag2_group_id], map_id: well_index)` will bring back the tags in the correct order in the list (tag 1 first, tag 2 second).

It makes me a bit nervous though as we're relying on the order of the query results. 

We could split `tags` into two methods, `tag1` and `tag2`, which would make sure it's deterministic? So `tag1` would do `Tag.where(tag_group_id: tag_group_id, map_id: well_index)`, for instance.